src/backend/web/routes/feeds.js

```diff
@@ -51,4 +51,26 @@ feeds.get('/:id', async (req, res) => {
   }
 });
 
+feeds.post('/', async (req, res) => {
+  const feedData = req.body;
+
+  try {
+    const feed = await Feed.create(feedData);
```

Comment:
Let's check if this feed already exists in our system before we try to add it again:

```js
if(await Feed.byUrl(feedData.url)) {
  res.status(409).json({message: `Feed for url ${feedData.url} already exists.`});
  return;
}
```